### DISCUSSION TITLE
Что добавить в тех. радар?

### DISCUSSION THREAD

**Discussion by osipxd** [RedMadRobot/techradar-android#11] [Ideas]
Тут всё, чего не хватает в радаре, но оно должно там быть.

Один комментарий — одна страница радара, в ветках обсуждения. Приоритетные страницы определяем количеством голосов.

**Comment by osipxd** [comment]
**Utilities (?):** Gradle Version Catalogs, вместе со ссылкой на наши каталоги

**Comment by sonulen** [comment]
описать сам тех радар и зачем он нам

**Comment by osipxd** [comment]
**Methods & Patterns:** Ручной DI aka Dagger-free
